app/index.js

```diff
@@ -494,21 +504,33 @@ app.on('ready', () => {
         return
       }
 
-      // TODO: If the username already exists, s/save/update
       var message = username
-        ? 'Would you like Brave to save the password for ' + username + ' on ' + origin + '?'
-        : 'Would you like Brave to save this password on ' + origin + '?'
-      dialog.showMessageBox({
-        type: 'question',
-        title: 'Save password?',
-        message: message,
-        buttons: ['Yes', 'No'],
-        defaultId: 0,
-        cancelId: 1
-      }, (buttonId) => {
-        if (buttonId !== 0) {
+        ? `Would you like Brave to remember the password for ${username} on ${origin}?`
+        : `Would you like Brave to remember your password on ${origin}?`
+
+      if (!(message in passwordCallbacks)) {
+        // Notification not shown already
+        appActions.showMessageBox({
+          buttons: ['Yes', 'No', 'Never for this site'],
```

Comment:
Sort of. "Always for this site" would mean that any password you enter on foo.com is always saved/updated automatically. I don't think this is a useful option because if someone enters a wrong password, this will permanently delete their previously-saved password.